libs/components/forms/testing/src/radio-fixture.ts

```diff
@@ -79,12 +79,14 @@ export class SkyRadioFixture {
     return this.debugEl.queryAll(By.css('.sky-radio-group sky-radio input'));
   }
 
-  private getRadioButtonInputEl(index: number): DebugElement {
+  private getRadioButtonInputEl(index: number): DebugElement | undefined {
     const allRadioInputs = this.getAllRadioInputEls();
 
     if (allRadioInputs && allRadioInputs[index]) {
       return allRadioInputs[index];
     }
+    /* istanbul ignore next */
```

Comment:
Possible to put `/* istanbul ignore else */` on the `if` statement, instead?

Reply:
Nope - I tried. Istanbul doesn't see it for some reason.